azure-devops/azext_devops/dev/team/credentials.py

```diff
@@ -39,8 +39,15 @@ def credential_set(organization=None):
         # Hence, handle anonymous user case here.
         if connection_data.authenticated_user.id == _ANONYMOUS_USER_ID:
             raise CLIError("Failed to authenticate using the supplied token.")
-    set_credential(organization=organization, token=token)
-    _check_and_set_default_organization(organization)
+        try:
+            set_credential(organization=organization, token=token)
+        except Exception as ex:  # pylint: disable=bare-except
+            logger.warning("Unable to use secure credential store in this environment.")
```

Comment:
Out of curiosity - How are sentence splits decided? Can't it be one message?

Reply:
It will go across length of terminal, which will not look good.